release-notes/5.0/5.0-known-issues.md

```diff
@@ -175,15 +178,33 @@ The Virtualization component is new in RC1 and there are some fixes we didn't fi
             <InnerBuildPropertyValues>TargetFrameworks</InnerBuildPropertyValues>
         </PropertyGroup>
         ```
-4. When launching Blazor apps from VS or initializing a debugging session in the browser, port 9300 must be open on the machine the Blazor app is running under.
+4. Resolved -- When launching Blazor apps from VS or initializing a debugging session in the browser, port 9300 must be open on the machine the Blazor app is running under.
 
 ### RC2
 1. For Windows Application Packaging projects targeting net5.0, we recommend targeting the Windows SDK 19041.  This Windows SDK is not installed by default in the Visual Studio installer options though is available and an Optional component.
     - You might see build error `APPX3217`: `SDK folder containing 'UAP.props' for 'UAP 10.0.19041.0' could not be located.`
-2. Dotnet tool pack does not support the new net5.0 framework alias. Please continue to target netcoreapp3.1 for now for WPF or Winforms dotnet tools. See https://github.com/dotnet/sdk/pull/13924
-3. Publishing with ReadyToRunUseCrossgen2 fails with an `An assembly specified in the application dependencies manifest (crossgen2.deps.json) was not found` error. 
+2. Resolved -- Dotnet tool pack does not support the new net5.0 framework alias. Please continue to target netcoreapp3.1 for now for WPF or Winforms dotnet tools. See https://github.com/dotnet/sdk/pull/13924
+3. Resolved -- Publishing with ReadyToRunUseCrossgen2 fails with an `An assembly specified in the application dependencies manifest (crossgen2.deps.json) was not found` error. 
     - A simple workaround is available here: https://github.com/dotnet/runtime/issues/43141#issuecomment-707249158. This issue should be resolved in .net RTM builds
-4. There is a known issue with the LazyAssemblyLoader service in Blazor as documented in https://github.com/dotnet/aspnetcore/issues/26631.
+4. Resolved -- There is a known issue with the LazyAssemblyLoader service in Blazor as documented in https://github.com/dotnet/aspnetcore/issues/26631.
+
+### GA
+1. Design time build failure and dependency node warning icons when targeting *netcoreapp2.1* and specifying a *RuntimeIdentifier* in the project file.  This is the default for Service Fabric projects.
+
+   **Error experience**
+
+   `Project is targeting runtime 'win7-x64' but did not resolve any runtime-specific packages for the 'Microsoft.NETCore.App' package. This runtime may not be supported by .NET Core.`
+
+   **Workaround**
+
+   Add a *Directory.Build.Targets* file with the following content
+
+   ```xml
+   <Project>
+     <Target Name="EnsureNETCoreAppRuntime" />
+   </Project>
+   ```
+2. Starting with .NET 5 preview 8, the .NET SDK requires Visual Studio 16.8 or newer. The SDK will not load in 16.7 and if you try to target `net5.0`, you will get compile errors. The error will be improved in a future release of 16.7 to advise you to update your Visual Studio installation. https://github.com/dotnet/msbuild/pull/5826
```

Comment:
@rbhanda We've had a bunch of confusion about .net 5 requiring 16.8.  I'm not sure the known issues list is the right place for this (it's listed in our installer but people don't notice that).  Is there a better spot for that to be documented?

Reply:
We will have release notes for 5.0 so an equivalent of 5.0 release notes like https://github.com/dotnet/core/blob/master/release-notes/5.0/preview/5.0.0-rc.2.md#visual-studio-compatibility seems like the best place for this documentation.